### ISSUE TITLE
Bug: ESC panel isn't working properly

### ISSUE THREAD

**Issue by bnickodemus** [dronecan/gui_tool#55]
The ESCs I'm using require a throttle signal of 40Hz-490Hz. The DroneCAN ESC panel sends throttle messages at 10Hz (0.10 sec). Changing the broadcast interval down to 0.01 sec doesn't increase the frequency of uavcan.equipment.esc.RawCommand packets. Note: I'm using DroneCAN GUI Tool v1.2.25 on windows.
![dronecan](https://github.com/user-attachments/assets/cd9434a5-792d-4c7e-9883-2e2d5485930b)

I then test using UAVCAN GUI Tool V1.0.0 on windows. I open the ESC panel and set the broadcast interval to 0.01 sec (100Hz). Then I go to the Bus Monitor, and I see that the frequency is 100Hz
![uavcan](https://github.com/user-attachments/assets/43523ac5-93b5-48f3-8e92-a4319fa651cc)

In summary, DroneCAN GUI Tool doesn't spin my motors, but UAVCAN GUI Tool works great.

If PX4 and ArduPilot firmware send RawCommand at 50Hz, why isn't 50Hz the default in the ESC Management Panel?

**Comment by tridge** [comment]
fixed in #60
